docs/auth/README.adoc

```diff
@@ -0,0 +1,123 @@
+= AeroGear Services Auth SDK
+
+Mobile authentication SDK based on link:http://www.keycloak.org/[Keycloak] using link:http://openid.net/connect/[OpenID Connect].
+
+== Usage
+
+To use the Auth SDK you'll first need to:
+
+* Have a Keycloak instance. See the <<Setting up Keycloak>> section.
+* Import the Core module. See link:./core/README.adoc[the Core documentation].
+* A configuration file added to the app directory. See the <<Configuration File>> section.
+
+=== Setting up Keycloak
+
+==== Using OpenShift
+
+* If you do not have mobile services enabled in your openshift cluster follow this link:https://github.com/aerogear/mobile-core/blob/master/docs/walkthroughs/local-setup.adoc[Local Setup] guide.
+* Navigate to your Openshift cluster and in the Service Catalog search for the Keycloak service.
+* Click on the Keycloak service and you will be prompted to fill in details about your app.  For now you can leave these as they are.  Navigate through the setup and click Create.
+This will provision the Keycloak service in the project you specified and create a public Client to be used with an app along with a bearer-only client.
+See link:http://www.keycloak.org/docs/latest/server_admin/index.html#oidc-clients[Keycloak OIDC client documentation].
+
+After provisioning, the Keycloak service will be available at the exposed Route. You can view this route inside your project or use the below command to get the route:
+----
+oc get route keycloak --template "http://{{.spec.host}}"
```

Comment:
This is no longer actual. People should use mobile-cli to get mobile-services.json file etc.